go/api/go_api/tests/utils.py

```diff
@@ -1,5 +1,27 @@
 from mock import Mock, patch
 
+from vumi.tests.utils import MockHttpServer
```

Comment:
I can take a stab at this if you like, although it would be nice to spread the knowledge around a bit. Either way, I'm happy to help.